src/api/app/controllers/webui/staging_workflows_controller.rb

```diff
@@ -54,7 +54,7 @@ def destroy
         render js: "window.location='#{project_show_path(@project)}'"
       else
         format.js { flash[:error] = "Staging Workflow for #{@project.name} couldn't be deleted: #{@staging_workflow.errors.full_messages.to_sentence}." }
-        render js: "window.location='#{redirect_back(fallback_location: root_path)}'"
+        render js: "window.location='#{root_path}'"
```

Comment:
Please squash this commit into the one that introduced this change. There is no need to have commits with interims changes, as long as it's not pushed to master yet.